acore/enrichment_analysis.py

```diff
@@ -308,7 +388,8 @@ def run_enrichment(
             num_background = num_background[0]
         else:
             num_background = 0
-        if method == "fisher" and num_foreground > 1:
```

Comment:
to discuss. should we mistrust enrichements which are based on single hits?

Reply:
I am not sure they would be significant but filtering them would reduce the number of tests, which may be good in terms of performance.